draft-ietf-rats-architecture.md

```diff
@@ -842,40 +842,47 @@ a different format.
 ~~~~
 {:multievidence #multievidence_diag title="Multiple Attesters and Relying Parties with Different Formats"}
 
-# Freshness {#freshness}
-
-It is important to prevent replay attacks where an attacker replays
-old Evidence or an old Attestation Result that is no longer correct.
-To do so, some mechanism of ensuring that the Evidence and Attestation
-Result are fresh, meaning that there is some degree of assurance
-that they still reflect the latest state of the Attester, and that any Attestation
-Result was generated using the latest Appraisal Policy for Evidence.
-There is, however, always a race condition possible in that the
-state of the Attester, and the Appraisal Policy for Evidence,
-might change immediately after the Evidence or Attestation Result was generated.
-The goal is merely to narrow the time window to something the Verifier
-(for Evidence) or Relying Party (for an Attestation Result) is willing to accept.
-
-There are two common approaches to providing some assurance of freshness.
-The first approach is that a nonce is generated by a remote entity
-(e.g., the Verifier for Evidence, or the Relying Party for an Attestation
-Result), and the nonce is then signed and included along with the claims in
-the Evidence or Attestation Result, so that the remote entity knows that the
-claims were signed after the nonce was generated.
-
-A second approach is to rely on synchronized clocks, and include a signed
-timestamp (e.g., using {{-rats-tuda}}) along with the
-claims in the Evidence or Attestation Result, so that the remote entity knows
-that the claims were signed at that time, as long as it has some assurance that
-the timestamp is correct.  This typically requires additional claims about
-the signer's time synchronization mechanism in order to provide such assurance.
-
-In either approach, it is important to note that the actual values in claims
-might have been generated long before the claims are signed.  If so, it is the
-signer's responsibility to ensure that the values are still correct when they
-are signed.  For example, values might have been generated at boot, and then
-used in claims as long as the signer can guarantee that they cannot have changed
```

Comment:
"could not have"

Reply:
Ned, you had another comment on this same sentence, which replaced it altogether.  Since your later suggestion has been already incorporated (c215569), I'm closing this.